internal/x/format/base_visitor.go

```diff
@@ -134,23 +134,45 @@ func (v *baseVisitor) POptions(isFieldOption bool, options ...*proto.Option) {
 			}
 		}
 		v.PComment(o.Comment)
-		switch len(o.AggregatedConstants) {
-		case 0:
+		// TODO: this is a good example of the reasoning for https://github.com/uber/prototool/issues/1
+		if len(o.Constant.Array) == 0 && len(o.Constant.OrderedMap) == 0 {
 			v.PWithInlineComment(o.InlineComment, prefix, o.Name, ` = `, o.Constant.SourceRepresentation(), suffix)
-		case 1:
-			v.PWithInlineComment(o.InlineComment, prefix, o.Name, ` = { `, o.AggregatedConstants[0].Name, ": ", o.AggregatedConstants[0].Literal.SourceRepresentation(), " }", suffix)
-		default:
+		} else if len(o.Constant.Array) > 0 { // both Array and OrderedMap should not be set simultaneously, need more followup with emicklei/proto
+			// TODO https://github.com/uber/prototool/issues/59
```

Comment:
Should we put a log statement here? or would this make too much noise?

Reply:
Will test out in a follow-up PR. Basically need to test how to parse:

```
extend google.protobuf.FieldOptions {
  repeated int64 custom_repeated_option = 80001;
}
```

I think it will be:

```
message Foo {
  string bar = 1 [(custom_repeated_option) = [1, 2, 3]];
}
```

But I am not sure.

Would not want to do logging personally.